src/components/draft-js-editor/__tests__/DraftJSEditor.test.js

```diff
@@ -75,7 +75,7 @@ describe('components/draft-js-editor/DraftJSEditor', () => {
             expect(editor.prop('ariaAutoComplete')).toEqual(inputProps['aria-autocomplete']);
             expect(editor.prop('ariaExpanded')).toEqual(inputProps['aria-expanded']);
             expect(editor.prop('ariaOwneeID')).toEqual(inputProps['aria-owns']);
-            expect(editor.prop('role')).toEqual(inputProps.role);
+            expect(editor.prop('role')).toEqual(inputProps.textbox);
```

Comment:
It appears that `inputProps.textbox` is undefined here. Can we hardcode the expected result and see if the tests passes?